resources/views/tasks/index.blade.php

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>task index</title>
+    <link rel="stylesheet" href="{{ asset('css/style.css') }}">
+</head>
+
+<body>
+    <h1>タスク一覧</h1>
+    <!-- <ul>-->
+
+    <div class="task-view" style="display: flex; flex-direction: column;">
+        @foreach ($tasks as $task)
+            <div style="display: flex; align-items: center; margin-bottom: -10px; margin-top: -15px;">
+                <!-- // リンク先をidで取得し名前で出力 -->
+                <p style="margin-right: 0px;"><a href="/tasks/{{ $task->id }}">{{ $task->title }}</a></p>
```

Comment:
動作は同じですが、routeヘルパーを使用するようにしましょう。
ぱっと見でのルーティングがわかりやすくなります！
他の箇所ではrouteヘルパを使用されているので、ここだけ書き間違いかもしれませんね。。
```suggestion
                <p style="margin-right: 0px;"><a href="{{ route('tasks.show', $task) }}">{{ $task->title }}</a></p>
```